Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -270,5 +278,25 @@ open class MessageContentCell: MessageCollectionViewCell {
 
         messageBottomLabel.frame = CGRect(origin: origin, size: attributes.messageBottomLabelSize)
     }
-    
+
+    /// Positions the cell's accessory view.
+    /// - attributes: The `MessagesCollectionViewLayoutAttributes` for the cell.
+    open func layoutAccessoryView(with attributes: MessagesCollectionViewLayoutAttributes) {
+        guard attributes.accessoryViewSize != .zero else { return }
+
+        let y = (bounds.height - attributes.accessoryViewSize.height) / 2
```

Comment:
Doesn't the `accessoryViewPadding` allow the user to adjust the y position here?

Reply:
Good point. Will update

Reply:
I'm wondering what this looks like when we have varying label sizes, avatar sizes, etc. What would this look like if the avatar size is bigger than the message? What if there is a very large label? Lots of things to think about